Doc/howto/functional.rst

```diff
@@ -528,19 +516,16 @@ Passing values into a generator
 -------------------------------
 
 In Python 2.4 and earlier, generators only produced output.  Once a generator's
```

Comment:
I think we could drop the versions here and just describe the state of the language in Python 3.x. Maybe drop the first paragraph with "there was no elegant way..." entirely, and the new first paragraph would be something like:
```
It's also possible to pass values into a generator. The previous examples have used `yield x` alone on a line, as if :keyword:`yield` is a statement, but in fact it's an expression that returns a value that can be assigned to ...
```